### PR TITLE
Sed festival decision and event chain for Kemetics #12

### PR DESCRIPTION
**Pull request by rquinio**
I've run the validator, and tested the events by commenting the timer and ruled_years conditions.
Tested that gluttonous + drunkard character sometime triggers failure but does not seem too often.

### CODE REVIEW

File: AncientReligions/decisions/ACR_egyptian_decisions.txt [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+				
+				# Can be renewed every 3 years
+				add_character_modifier = {
+					name = held_sed_festival_timer
+					duration = 1080
+					hidden = yes
+				}
+				
+				# Keep counter of number of Sed vestivals hold
+				change_variable = { which = nb_sed_vestivals value = 1 }
```
Comment (xirsoi) [inline AncientReligions/decisions/ACR_egyptian_decisions.txt]: typo ;) "vestivals"

File: AncientReligions/events/ACR_egyptian_events.txt [hunk 2]
```diff
@@ -329,4 +330,319 @@ narrative_event = {
...
+
+#Sed festival 2nd day: the race
+character_event = {
+	id = ancrel.4011
+	desc = "EVTDESC_ancrel_4011"
+	picture = "GFX_evt_temple_egyptian_pagan"
+	
+	is_triggered_only = yes
+	
+	immediate = {
```
Comment (xirsoi) [inline AncientReligions/events/ACR_egyptian_events.txt]: You have a couple of these empty immediate clauses, are they necessary to keep Validator quiet?

File: AncientReligions/events/ACR_egyptian_events.txt [hunk 3]
```diff
@@ -329,4 +330,319 @@ narrative_event = {
...
+		
+	}	
+
+	option = {
+		name = "EVTOPTA_ancrel_4011" # OK
```
Comment (xirsoi) [inline AncientReligions/events/ACR_egyptian_events.txt]: You should remove the quotes from your localization keys.

Comment (rquinio) [inline AncientReligions/events/ACR_egyptian_events.txt]: Yeah, I agree. CK2+ used it, but vanilla doesn't.

File: AncientReligions/events/ACR_egyptian_events.txt (hunk 2 above)
Comment (rquinio) [inline AncientReligions/events/ACR_egyptian_events.txt]: Nope, will remove

**Comment by rquinio** [comment]
I'll redo the PR from a feature branch from repo.

**Pull request closed without merging**